src/components/incoming-data-menu/incoming-data-menu.html

```diff
@@ -1,86 +1,111 @@
 <action-sheet>
   <ion-list *ngIf="type === 'url'">
-    <ion-item-divider>{{'Website' | translate}}</ion-item-divider>
+    <ion-item-divider>{{ 'Website' | translate }}</ion-item-divider>
     <ion-item>
-      {{data}}
+      {{ data }}
       <ion-icon item-start>
-        <img *ngIf="!https" src="assets/img/icon-lock-x.svg">
-        <img *ngIf="https" src="assets/img/icon-lock-green.svg">
+        <img *ngIf="!https" src="assets/img/icon-lock-x.svg" />
+        <img *ngIf="https" src="assets/img/icon-lock-green.svg" />
       </ion-icon>
     </ion-item>
     <button ion-item (click)="close('OpenExternalLink', data)">
       <ion-icon item-start>
-        <img src="assets/img/icon-link-external.svg">
+        <img src="assets/img/icon-link-external.svg" />
       </ion-icon>
       <span translate>Open website</span>
     </button>
     <button ion-item copy-to-clipboard="{{ data }}">
       <ion-icon item-start>
-        <img src="assets/img/icon-paperclip.svg">
+        <img src="assets/img/icon-paperclip.svg" />
+      </ion-icon>
+      <span translate>Copy to clipboard</span>
+    </button>
+  </ion-list>
+
+  <ion-list *ngIf="type === 'InvoiceUrl'">
```

Comment:
Added incoming-data-menu for invoice.

<img width="272" alt="Screen Shot 2019-04-24 at 5 19 16 PM" src="https://user-images.githubusercontent.com/23103037/56694645-25c08b00-66b5-11e9-87b3-cca6947983c6.png">